mediawiki_matrix_bot/__init__.py

```diff
@@ -124,17 +125,12 @@ async def forward_news(client, room, message_obj,baseurl):
         )
 
 
-async def fetch_changes(baseurl):
+async def fetch_changes(baseurl: str) -> Any:
     async with aiohttp.ClientSession() as session:
         async with session.get(f'{baseurl}/api.php?action=query&list=recentchanges&format=json&rcprop=user|comment|flags|title|sizes|loginfo|ids|revision') as response:
-            return await response.json()
```

Comment:
"official" documentation contained the await, not sure if it is actually required

Reply:
https://docs.aiohttp.org/en/stable/client_quickstart.html

Reply:
Making the function async and then Not awaiting a blocking call makes little sense.

Reply:
Ah, sorry. I was trying to fix the type but than forgot to re-add it again.